repos/rust-lang/this-week-in-rust.toml

```diff
@@ -0,0 +1,12 @@
+org = "rust-lang"
+name = "this-week-in-rust"
+description = "Data for this-week-in-rust.org"
+bots = []
+
+[access.teams]
+twir-reviewers = "maintain"
+twir = "maintain"
```

Comment:
Same point as the other PR about the need for maintain - `admin` or `write` should be enough

Reply:
Changed. I think that `maintain` can be used to push directly to protected branches, I wonder if that is used on some of these repos where CI is not set up and/or PRs are not used so commonly.

Reply:
You're right. I also realized that I was switching `maintain` and `admin` in my head. So `maintain` actually likely is the appropriate permission in cases where we want to allow the maintainers to more-or-less do their own thing while `admin` is likely never appropriate.

Reply:
Yeah, I think that `maintain` is/was used in the cases where the teams/contributors had a large degree of autonomy over the repo. That being said, I think that it's not a bad idea to proactively lower the permissions to `write` and only allow `maintain` if someone really requires it.